mkosi/resources/mkosi-initrd/mkosi.conf.d/10-serpentos.conf

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=serpentos
+
+[Content]
+# Completely override the default Packages
+Packages=
+        dash # for /bin/sh
+        # Various libraries that are dlopen'ed by systemd
+
+        # File system checkers for supported root file systems
+        e2fsprogs
+        xfsprogs
+
+        # fsck.btrfs is a dummy, checking is done in the kernel.
+
+        util-linux
+
+RemoveFiles=
+        /.moss
+
+        # Remove all files that are only required for development.
+        /usr/lib/*.a # static libs
+        /usr/lib/gconv # encoding conversion
+        /usr/lib/kernel # kernelinstall files
```

Comment:
There is `installkernel` and `kernel-install`, but no `kernelinstall` that I know of. Typo?